### DISCUSSION TITLE
在简悦的阅读模式下使用翻译功能的几种方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1561] [使用技巧]
**可以屏幕取词的 App 都可以实现**。

1. 可以桌面装欧陆词典，直接屏幕取词并加入生词本（欧陆全平台同步
2. [Bob](https://sspai.com/post/62721)

***

也可以使用 [沙拉查词的浏览器外取词](https://saladict.crimx.com/native.html) 方式

***

如果对翻译不是要求很高的话，也可以使用简悦自带的两款插件：

1. [划词搜索](https://simpread.ksria.cn/plugins/details/ohnTKVHz4a)

2. [全文翻译](https://simpread.ksria.cn/plugins/details/Y7JxbP7B4H)

**Comment by Kenshin** [comment]
现在提供另外一种全文翻译的思路（以彩云小译为例，当然也可以将其替换为其它方案）。

1. 先将某个站加入到延迟加载列表
2. 用彩云小译
3. 进入阅读模式
4. 重新高亮

这时，就能得到下图（下图为中英译文，也可以随意调整为全部中文

![image](https://user-images.githubusercontent.com/81074/114294812-9d229b80-9ad3-11eb-8d78-4d39c91d0b03.png)

***

注意：新版（未发布）会优化/降低第四步的操作时长和流程。

### 延伸阅读

- [延迟加载](http://ksria.com/simpread/docs/#/%E8%AF%8D%E6%B3%95%E5%88%86%E6%9E%90%E5%BC%95%E6%93%8E?id=%e5%bb%b6%e8%bf%9f%e5%8a%a0%e8%bd%bd)

**Reply by YangWaldon** [reply]
哇塞，这个太好啦，太谢谢啦
另外关于简悦的标注功能，我想提几个建议，
1、比如有时候我在一段话里有好几个高亮，当我想要对某个标注打标签的时候，那个自动显示的标注栏总是来回跳转，很难选中，除非开启连续标注进行鼠标双击，
2、建议复制功能支持ctrl+v
3、建议支持annotion或者打标签的快捷键，还有输入完成之后退出编辑界面的快捷键，比如esc,要不然总是在鼠标或者键盘之间切换效率很低

**Reply by YangWaldon** [reply]
另外再请教一个问题，进入阅读模式后，做的高亮或者标注都是自动加入到稍后读的嘛，我在高级选项和帮助文档里没找到相关说明

**Reply by Kenshin** [reply]
@YangWaldon 

嗨，我把你的问题转移到 https://github.com/Kenshin/simpread/issues/2042 并进行了回复。 😀

**Reply by YangWaldon** [reply]
今天取消了简悦的预加载模式，按照上文您的教程，确实实现了，但是想请教下，就您所说的随意切换中英文或者原文，是需要退出阅读模式进行切换操作后再进入阅读模式吗

**Comment by jiangaq** [comment]
使用 [imTranslator](https://chrome.google.com/webstore/detail/imtranslator-translator-d/noaijdpnepcgjemiklgfkcfbkokogabh) 的 **inline translator** 功能，可以在简悦的阅读模式下使用。

**Reply by YangWaldon** [reply]
又发现一个神器哈哈哈，多谢啦，这个太实用啦

**Reply by YangWaldon** [reply]
我今天安装了这个插件，发现翻译全文的功能无法使用，请问您指导怎么解决嘛，然后在网页上使用简悦浏览期刊文章时，经常匹配不成功

> 使用 [imTranslator](https://chrome.google.com/webstore/detail/imtranslator-translator-d/noaijdpnepcgjemiklgfkcfbkokogabh) 的 **inline translator** 功能，可以在简悦的阅读模式下使用。

**Reply by jiangaq** [reply]
是用的谷歌引擎，如果没T那可以改成Bing引擎试试

**Reply by YangWaldon** [reply]
> 是用的谷歌引擎，如果没T那可以改成Bing引擎试试

emmm,,其他几个选项都找到了可以更换翻译引擎的选项，这个全文翻译好像无法更换翻译引擎

**Reply by YangWaldon** [reply]
> 是用的谷歌引擎，如果没T那可以改成Bing引擎试试

而且在进入简悦的阅读模式后，我这里是没办法使用inline translator的功能的，请问您知道问题出在哪里了嘛

**Reply by Kenshin** [reply]
嗨，各位。

更新提示，[全文翻译](https://github.com/Kenshin/simpread/discussions/1593) 更新到了 1.1.0 增加了下面的新功能：

- `Add` [进入阅读模式后自动启用与白名单](https://github.com/Kenshin/simpread/discussions/1593#discussioncomment-3444778)

- `Add` [段落翻译](https://github.com/Kenshin/simpread/discussions/1593#discussioncomment-3444569)

- `Add` [DeepL](https://github.com/Kenshin/simpread/discussions/1593#discussioncomment-3492095)

**Comment by Kenshin** [comment]
关联 https://github.com/Kenshin/simpread/issues/317#issuecomment-845887075
